toolchain/driver/compile_subcommand.cpp

```diff
@@ -443,7 +443,7 @@ class CompilationUnit {
     sem_ir_converter_.emplace(node_converters, &*sem_ir_);
     return {.consumer = consumer_,
             .value_stores = &value_stores_,
-            .timings = &timings_,
+            .timings = timings_ ? &*timings_ : nullptr,
```

Comment:
[abseil.io/tips/141](https://abseil.io/tips/141) makes a case in favor of explicit `.has_value()` for clarity

(I'm a new contributor, LMK if there's already a style decision on this!)

Reply:
I believe that is more for optional<T> where T converts to bool, as it becomes more ambiguous.

Reply:
FWIW I think we don't generally use that tip. i.e., we don't often write `!= nullptr` or `.has_value()`. We do generally avoid implicit condition checks for ints (`!= 0`) and enums.

i.e., as-is looks good.